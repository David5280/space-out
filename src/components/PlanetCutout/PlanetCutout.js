import React from 'react';
import { NavLink } from 'react-router-dom';

const PlanetCutout = ({ cutout, title }) => {
  return (
    <article className={`single-article ${title}-cutout`}>
      <NavLink to={`${title}`} >
        <img 
          src={cutout} 
          alt={title} 
          className='planet-home-image'
        />
      </NavLink>
    </article>
  )
}

export default PlanetCutout;