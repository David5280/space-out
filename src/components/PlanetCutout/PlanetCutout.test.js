import React from 'react'
import PlanetCutout from './PlanetCutout';
import { shallow } from 'enzyme';

describe('PlanetCutout', () => {
  it('should match the snapshot', () => {
    const props = {
      title:  'earth',
      cutout: 'earth image',
      key: 4
    }
    const wrapper = shallow(<PlanetCutout {...props}/>)
    expect(wrapper).toMatchSnapshot();
  })
})