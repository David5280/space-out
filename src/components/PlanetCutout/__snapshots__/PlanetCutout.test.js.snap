// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PlanetCutout should match the snapshot 1`] = `ShallowWrapper {}`;
